README.md

```diff
@@ -14,7 +14,7 @@ This is the main Exonum repository that includes
 * [Exonum testing framework](https://github.com/exonum/exonum/blob/master/testkit/README.md).
 * Services:
   * Configuration service (currently in [exonum-configuration](https://github.com/exonum/exonum-configuration) repository).
-  * Time service (currently in [exonum-time](https://github.com/exonum/exonum-time) repository).
+  * [Time service](https://github.com/exonum/exonum/blob/master/services/exonum-time/README.md).
```

Comment:
Nitpick: I think it could make sense to use a relative link here: `[Time service](services/exonum-time/README.md)`.

Reply:
There is (or was?..) a problem with relative links in the crates.io, but that isn't applied to the root readme.